### PR TITLE
Fix issue101 - Localize Thank you page

### PR DESCRIPTION
**Pull request by alicoding**
This patch will only make sure to display the form for English version only.

### CODE REVIEW

**Comment by alicoding** [comment]
@ScottDowne R?

File: components/form.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	    }
+	  });
+	},
+	render: function() {
+		return (
+			<div>
+				<div className="form-wrapper container">
+				  <div className="wrap">
+				    <div className="row">
+				      <h2>Sign up for email updates</h2>
```
Comment (ScottDowne) [inline components/form.jsx]: What's with the whitespace?

File: components/form.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
+var React = require('react');
```
Comment (ScottDowne) [inline components/form.jsx]: Can we call this signup.jsx instead of form.jsx?

**Comment by ScottDowne** [comment]
Just change the name of form.jsx and hit merge.

**Pull request merged**